packages/service-library/src/servicelib/background_task.py

```diff
@@ -21,21 +22,21 @@ async def _periodic_scheduled_task(
     interval: datetime.timedelta,
     task_name: str,
     **task_kwargs,
-):
+) -> None:
```

Comment:
Use ``NoReturn`` instead of ``None``

SEE https://peps.python.org/pep-0484/#the-noreturn-type